nilrt_snac/_configs/_graphical_config.py

```diff
@@ -8,15 +8,17 @@
 
 
 class _GraphicalConfig(_BaseConfig):
-    """The graphical configuration for SNAC is to deconfigure the X11, embedded UI, and other components that are only useful when using the graphical UI.
-    """
+    """The graphical configuration for SNAC is to deconfigure the X11, embedded UI, and other components that are only useful when using the graphical UI."""
```

Comment:
Since you're touching this line...
```suggestion
    """The graphical configuration for SNAC is to deconfigure the X11, embedded UI, and other components that are useful only when using the graphical UI."""
```